amplify/backend/function/risestandstrongcb34053a/src/routes/shifts.js

```diff
@@ -198,7 +199,18 @@ router.get('/:startTimestamp', async (req, res) => {
  *            description: startTimestamp not found
  */
 router.put('/:startTimestamp', async (req, res) => {
-   res.end();
+   let startTimestamp = req.params.startTimestamp;
+   let body = {
```

Comment:
Same thing with the body
```suggestion
   const body = {
```